docs/en/enterprise-edition/policy-reference/aws-policies/s3-policies/bc-aws-s3-24.adoc

```diff
@@ -1,4 +1,4 @@
-== AWS S3 bucket is not configured with MFA Delete
+== S3 bucket policy allows lockout all but root user
```

Comment:
Options:
- S3 bucket policy restricts access to root user only
- S3 bucket policy denies access to non-root users